.eslintrc.yml

```diff
@@ -30,7 +32,7 @@ rules:
   no-warning-comments: off
   prefer-const: error
   mozilla/no-import-into-var-and-global: off
-  prettier/prettier: off
   comma-dangle: off
   react-hooks/rules-of-hooks: error
   react-hooks/exhaustive-deps: warn
+  prettier/prettier: warn
```

Comment:
Instead of skipping files in .gitignore, it looks like you've changed it so it's all just warnings. We want to give errors for bad formatting in normal source code, just not generated files (which are mostly listed in .gitignore).